crates/teloxide-macros/src/attr.rs

```diff
@@ -15,31 +15,59 @@ pub(crate) fn fold_attrs<A, R>(
     f: impl Fn(A, R) -> Result<A>,
 ) -> Result<A> {
     attrs
-        .iter()
-        .filter(|&a| filter(a))
+    .iter()
+    .filter(|&a| filter(a))
```

Comment:
this is broken formatting, please undo..

Reply:
Undo?

Reply:
Ohh, how that format was passed?

Reply:
we still use old rustfmt which dies on let else